.Rprofile

```diff
@@ -1,7 +1,8 @@
 if (Sys.getenv("GITHUB_ACTIONS") == "") {
   source("renv/activate.R")
-  envsetup::rprofile(config::get(file = file.path(getwd(), "_envsetup.yml")))
+  envsetup::rprofile(config::get(file = file.path(getwd(), "_envsetup.yml"), config = "prod"))
   source("inst/startup.R")
 } else {
   options(repos = "https://cran.microsoft.com/snapshot/2022-11-01")
+  envsetup::rprofile(config::get(file = file.path(getwd(), "_envsetup.yml"), config = "prod"))
```

Comment:
Please remove the new line in the `else` statement as this will make the workflows fail again.

Reply:
updated